### ISSUE TITLE
Sweep error with 0.9.0 

### ISSUE THREAD

**Issue by dangunter** [watertap-org/watertap-ui#88]
## What I did
- Downloaded and installed latest 0.9.0 release
- Opened the AMO 1690 flowsheet
- Did a little experimentation with solve/compare, then reset to default and changed to "SWEEP" mode
- Set "cloth media filtration" -> "water recovery" range between 0.85 and 0.95 
![image](https://github.com/watertap-org/watertap-ui/assets/420923/4bffa54d-80e0-45f5-82ec-0ae642c9d71e)
- Clicked on "RUN" button

## What happened
- Almost immediately got a popup error, saying "error : unknown sampling type None"
![image](https://github.com/watertap-org/watertap-ui/assets/420923/4cee131a-df1f-4664-83d0-e3c3ac4becbe)
- Looked at the logs, which report the involved Python line numbers:
```
[INFO] 2023-07-04 10:04:25,946 idaes.app.routers.flowsheets (flowsheets.py:128): Loading new data into flowsheet 'watertap.examples.flowsheets.case_studies.wastewater_resource_recovery.amo_1690.amo_1690_ui'
[INFO] 2023-07-04 10:04:25,947 idaes.app.routers.flowsheets (flowsheets.py:147): trying to sweep
[INFO] 2023-07-04 10:04:25,947 idaes.app.internal.parameter_sweep (parameter_sweep.py:85): trying to sweep
[ERROR] 2023-07-04 10:04:25,948 idaes.app.internal.parameter_sweep (parameter_sweep.py:163): err: Unknown sampling type: None
```

**Comment by MichaelPesce** [comment]
@dangunter it looks like you set the variables as free? To run the sweep, variables need to be set as sweep. We wanted to allow for variables to be set as free without being included in the sweep.
That being said, maybe it makes sense to disable the "run sweep" button if no variables have been set to sweep.

**Comment by dangunter** [comment]
oh that's confusing. yes, it shouldn't try to run a sweep if there are no variables to sweep. Some sort of popup to that effect would be useful.

**Comment by MichaelPesce** [comment]
fixed in #89
